python/large_test_files.py

```diff
@@ -95,33 +125,7 @@ def new_lar_rows(self, row_count=None):
         
         """Applies the unique_uli function to the new LAR dataframe 
         to generate a unique set of ULIs."""  
-        new_lar_df = self.unique_uli(new_lar_df)
```

Comment:
What was the reason for pulling the unique_uli function out of the class? 

I think if it's going to be a stand alone function it should live with other stand alone functions or in a combined utility class. 

I'd like to have class scripts in their own .py files to simplify keeping track of code parts.

Let's either move this function to a utilities class, a function utility file, or leave it in the large files class for now. We can address its final home in the upcoming refactor.